io.github.eclab.edisyn.metainfo.xml

```diff
@@ -0,0 +1,114 @@
+      <li>Oberheim Matrix 6, 6R, and 1000 (Single and (for 1000) Global Modes)</li>
+      <li>PreenFM2</li>
+      <li>Red Sound DarkStar and DarkStar XP2</li>
+      <li>Roland D-110 (Tone and Multi Modes)</li>
+      <li>Roland JV-80 and JV-880 (Single, Multi, and Drum Modes)</li>
+      <li>Roland U-110</li>
+      <li>Roland U-20/U-220 (Single, Multi, and Drum Modes)</li>
+      <li>Sequential Prophet Rev2</li>
+      <li>Waldorf Blofeld and Waldorf Blofeld Keyboard (Single and Multi Modes, plus Wavetable uploading)</li>
+      <li>Waldorf Kyra (Single and Multi Modes)</li>
+      <li>Waldorf M</li>
+      <li>Waldorf Microwave II, XT, and XTk (Single and Multi Modes)</li>
+      <li>Waldorf Pulse 2</li>
+      <li>Waldorf Rocket</li>
+      <li>Yamaha DX7 Family (DX7, TX7, TX802, TX216/TX816, Korg Volca FM, Dexed, DX200, DX9)</li>
+      <li>Yamaha 4-Op FM Family (DX21, DX27, DX100, TX81Z, DX11, TQ5, YS100, YS200, B200, V50, etc.) (Single and (for TX81Z and DX11) Multi Modes)</li>
+      <li>Yamaha FB01 (Single and Multi Modes)</li>
+      <li>Yamaha FS1R (Voice, Performance, and Fseq Modes)</li>
+      <li>Yamaha TG33, SY22, and SY35 (Single and (for TG33) Multi Modes)</li>
+      <li>General CC, NRPN, and RPN editing</li>
+      <li>Microtuning editing</li>
+    </ul>
+    <p>Edisyn has infinite levels of undo, CC and NRPN mapping and learning, offline modes, per-parameter customization, real-time parameter updates, test notes and chords, Pseudo-MPE support, and lots more.  Edisyn also has many specialized tools designed to help you explore new patch possibilities without directly programming them.  These include:</p>
+    <ul>
+      <li>
+        <em>Randomization:</em> Weighted patch mutation</li>
+      <li>
+        <em>Merging:</em> Weighted recombination of two patches of your choice</li>
+      <li>
+        <em>Mixing:</em> Bulk recombination of many patches into a single one</li>
+      <li>
+        <em>Blending:</em> Random recombination of two randomly-chosen patches on your synth</li>
+      <li>
+        <em>Nudging:</em> Pushing the patch to sound a bit more (or a bit less) like one of four other patches of your choice</li>
+      <li>
+        <em>Morphing:</em> Real-time interpolation of four patches to form a new patch</li>
+      <li>
+        <em>Hill-Climbing and Constriction:</em> Evolutionary techniques for guided randomized search through the space of parameters, where Edisyn iteratively offers patch possibilities for you to grade, then looks for new ones based on your assessments.</li>
+      <li>
+        <em>Deep-Learned Models:</em> Improvements to Randomization and Hill-Climbing using a deep-learned Variational Autoencoder neural network (DX7 Family only)</li>
+    </ul>
+
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Screenshot</caption>
+      <image type="source">https://raw.githubusercontent.com/eclab/edisyn/master/resources/pics/Banner.png</image>
```

Comment:
this should be from a commit not `master`

Reply:
considered in https://github.com/flathub/flathub/pull/5762/commits/4cb83eed7cc40189ceac5437a087392188ca1ce0